.github/workflows/master.yml

```diff
@@ -0,0 +1,44 @@
+name: master
```

Comment:
Unfortunately, we got to use the Drone CI only with docker credentials due to the infra team requirements.